branding/css/susemanager-tables.less

```diff
@@ -19,6 +19,10 @@ background-color: @table-bg-accent !important;
   vertical-align: middle !important;
 }
 
+.list-row-odd {
+  background-color: @gray-lighter;
```

Comment:
After doing some serious archeology work in uyuni less/css, it seems that the other striped tables (system list, for instance) use `@table-bg-accent`, not `@gray-lighter`. Or am I wrong?

Reply:
You are absolute right. I made the adjustment. TBH I could not make out a visual difference between the two (there is one though). Thanks for pointing that out!